### PR TITLE
Update QUEST.tsv [Lines 1801-1950]

### PR DESCRIPTION
**Pull request by ghost**
Lines 1801-1950 reviewed. Open to feedback.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -1800,154 +1800,154 @@ QUEST_20150406_001799	$Modestas admitted defeat and said he'd pay him pay back.
 QUEST_20150406_001800	Lost Magic Circle
 QUEST_20150414_001801	What do you want to know? 
 QUEST_20150414_001802	Receptionist Ramda
-QUEST_20150414_001803	Can start the commission when all party members are gathered. 
+QUEST_20150414_001803	$The commission can start once all party members have gathered. 
 QUEST_20150414_001804	You are not eligible to receive commissions yet. 
 QUEST_20150414_001805	Star commission?
```
Comment (emailboxu) [inline QUEST.tsv]: "Start commission?"

Missing a letter. You can $ this line.

**Comment by emailboxu** [comment]
Any particular reason you skipped a bunch of lines between your edits (didn't put in a $ on them)?  I was going to go through and see if they were $-able but I figured there was a reason for it.

**Comment by ghost** [comment]
Because I didn't do anything to change them because I thought they were fine but figured others in the future might think differently and would want to change them. Should I mark them anyway?

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**